doc/introduction.rst

```diff
@@ -0,0 +1,103 @@
+Introduction
+============
+
+Simpleline is a text UI framework. Originally a part of the Anaconda installer project.
+
+This UI is simple and easy to use. It is designed to be used with line-based machines and tools
```

Comment:
`This UI is simple and easy to use.` Well, you know my feelings on this statement from when I last pointed it out. I'm just pointing it out again here.